pom.xml

```diff
@@ -64,20 +64,22 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <oh.java.version>11</oh.java.version>
+    <oh.java.version>17</oh.java.version>
+    <oh.model.java.version>11</oh.model.java.version>
     <maven.compiler.source>${oh.java.version}</maven.compiler.source>
     <maven.compiler.target>${oh.java.version}</maven.compiler.target>
     <maven.compiler.compilerVersion>${oh.java.version}</maven.compiler.compilerVersion>
 
-    <bnd.version>6.3.0</bnd.version>
+    <bnd.version>6.4.0</bnd.version>
```

Comment:
I think we can upgrade Bnd today and do not have to wait for Java 17 to do this.